.github/workflows/deploy.testnet.yml

```diff
@@ -1,180 +0,0 @@
----
-name: TestNet Workflow
```

Comment:
Ultimately, we should consider reviving this workflow, which would spin up a testnet with the new software version and perform a set of sanity check tests that you implemented on each release or even on-demand.